hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAMetadataOnly.java

```diff
@@ -318,32 +318,34 @@ public void testReadRequest() throws Exception {
     }
   }
 
-  @Flaky("This test randomly failing. Let's enable once its fixed.")
   @Test
   public void testListVolumes() throws Exception {
     String userName = UserGroupInformation.getCurrentUser().getUserName();
     String adminName = userName;
     ObjectStore objectStore = getObjectStore();
 
+    String prefix = "vol-" + RandomStringUtils.randomNumeric(10) + "-";
+    VolumeArgs createVolumeArgs = VolumeArgs.newBuilder()
+        .setOwner(userName)
+        .setAdmin(adminName)
```

Comment:
I think you just pulled this up from the loop, but I guess the ``adminName`` is redundant with userName.
Could have dropped ``String adminName = userName;`` and just used userName instead...
Just sharing to see if I didn't misunderstood. :)